src/model/Entity/EntityFactory.php

```diff
@@ -1,13 +1,18 @@
 <?php
-namespace base\model;
+namespace base\model\entity;
 
 class EntityFactory
 {
 	protected $directory = "model";
 	
 	public function getEntity( $name )
 	{
-		$class = "project\\".$this->directory."\\".$name;
+		if(file_exists('./project/'.$this->directory.'/'.$name.'.php')){
```

Comment:
For an entityFactory, the purpose is to load an entity but where do u check if entity is really an entity? 
Here, I can inject an existing file, with a valid class but not specially an entity
Try is_a($class, EntityInterface, true)